doc/api/errors.md

```diff
@@ -1355,6 +1355,12 @@ The [`server.listen()`][] method was called while a `net.Server` was already
 listening. This applies to all instances of `net.Server`, including HTTP, HTTPS,
 and HTTP/2 Server instances.
 
+<a id="ERR_SERVER_NOT_RUNNING"></a>
+### ERR_SERVER_NOT_RUNNING
+
+The [`server.close()`][] method was called when a `net.Server` was not running. This applies to all instances of `net.Server`, including HTTP, HTTPS,
+and HTTP/2 Server instances.
```

Comment:
Nit: "Server" should probably be either in backticks or else lowercase. (Either it is referring to a server generically, in which case lowercase, or it is referring to the `Server` object, in which case backticks.)